src/libraries/System.Security.AccessControl/src/System.Security.AccessControl.csproj

```diff
@@ -83,6 +83,7 @@
     <Reference Include="System.Collections" />
     <Reference Include="System.Collections.NonGeneric" />
     <Reference Include="System.Diagnostics.Debug" />
+    <Reference Include="System.Memory" />
```

Comment:
This line already exists two lines below here (according to my GitHub diff view).

Reply:
Oh, hah. Missed that Aaron already did this in my rebase/merging adventures.